.gitignore

```diff
@@ -3,5 +3,9 @@
 .idea/tasks.xml
 .idea/**/uiDesigner.xml
 .idea/**/libraries
+.idea/inspectionProfiles/
+
 target/
+application.properties
```

Comment:
I've made it that way, because it was the simplest way to .gitignore confidential data without having to create separate file and then linking it in some way to application. properties. Theres is application.template.properties file which isn't ignored and contains config data without credentials.

Reply:
Right now I will leave it as it is. We can think about some other way of doing this in next task. :D